### DISCUSSION TITLE
Welcome to sammo Discussions!

### DISCUSSION THREAD

**Discussion by t-schn** [microsoft/sammo#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. This forum is a place to 
  * Ask questions you may have around using SAMMO
  * Brainstorm new features
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dxandtom** [comment]
when i use APO,it will happen:AttributeError: 'Output' object has no attribute 'with_parent'. Did you mean: '_sym_parent'?
And i found in search.py the "result" in "task" is output ,which is different from using "Paraphrase"
